src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2508,4 +2508,16 @@ Uninstall the following packages: {0}?</value>
   <data name="GetStartedGuideSurveyTitle" xml:space="preserve">
     <value>Finished</value>
   </data>
+  <data name="UseDefaultValuePopupMenuItem" xml:space="preserve">
+    <value>Use Default Value</value>
+  </data>
```

Comment:
I think we already have a resource `PortViewContextMenuUserDefaultValue` which is used in the old popup menu

Reply:
Sure, reinstated the old resource.